### ISSUE TITLE
[Resolved] WhatsApp works with Google Chrome 60+

### ISSUE THREAD

**Issue by rafatosta** [zapzap-linux/zapzap#1]
![image](https://github.com/zapzap-linux/zapzap/assets/18619894/e0c3f6ac-6125-46a5-95a1-2a7037d56e84)

### **Solution: Press F5.**

**Contextualization:**
This is an old problem that happens with WhatsappWeb, QtWebEngine and UserAgent. I never understood the reason, but if I refresh the page it works again. I used a strategy of reloading the page and force loading from the cache, but sometimes it happens.

**Related problems (some):**
- https://github.com/qutebrowser/qutebrowser/issues/7199
- https://github.com/qutebrowser/qutebrowser/issues/6812
- https://github.com/orkestral/venom/issues/780
- https://github.com/qutebrowser/qutebrowser/issues/4445
- https://github.com/qutebrowser/qutebrowser/issues/5210

**Comment by iitdwtf** [comment]
Yes this error page comes sometimes but upon closing and reopening the app it vanishes and the application runs normally. weird though.

**Comment by nickfera** [comment]
F5 did not help me. I disconnected the previous one from my phone, added a new account and connected the new one and it worked, but the first account is still showing the message.

**Comment by rafatosta** [comment]
> F5 did not help me. I disconnected the previous one from my phone, added a new account and connected the new one and it worked, but the first account is still showing the message.

There are times when refreshing the page doesn't help. It's happened to me a few times. 
You need to close it completely and open it again.

I will think of another approach to alleviate this problem urgently. Thanks for reporting

**Comment by KarthiDreamr** [comment]
> F5 did not help me. I disconnected the previous one from my phone, added a new account and connected the new one and it worked, but the first account is still showing the message.

same issue for me too

**Comment by jasjastone** [comment]
Hi,

> F5 did not help me. I disconnected the previous one from my phone, added a new account and connected the new one and it worked, but the first account is still showing the message.

Yes this help me solve the issue, but i think the issue is caused by the update made by whatsapp web it's self since i have add another account and i login successfully then it say's click to update whatsapp web so probably it is caused by the update whatsapp made
